components/Player/PlayerVideo/PlayerVideoPanel/PlayerVideoPanel.vue

```diff
@@ -58,8 +76,8 @@
 				</div>
 			</div>
 			<div class="buttons">
-				<SoftButton v-tooltip:bottom="t.bonus_point" icon="thumb_up" class="button-like" @click="counts.rating++" />
-				<SoftButton v-tooltip:bottom="t.minus_point" icon="thumb_down" class="button-dislike" @click="counts.rating--" />
+				<SoftButton v-tooltip:bottom="t.bonus_point" icon="thumb_up" class="button-like" @click="Upvote()" />
```

Comment:
no need to call the function, here and below
```suggestion
				<SoftButton v-tooltip:bottom="t.bonus_point" icon="thumb_up" class="button-like" @click="Upvote" />
```